syncto/views/errors.py

```diff
@@ -18,11 +19,8 @@ def response_error(context, request):
                              context.response.reason,
                              context.response.text)
 
-    statsd = request.registry.statsd
-
-    if statsd:
-        statsd.count("syncclient.status_code.%s" %
-                     context.response.status_code)
+    statsd_count(request, "syncclient.status_code.%s" %
+                 context.response.status_code)
```

Comment:
> Shouldn't we make sure the original error comes from Sync precisely?

This is a good point, also in Syncto we are using requests only to talk with the Sync infrastructure.
I would let it like that for now, plus it is not really related to that issue.  But I will add a XXX to make sure we do not forget about it.